src/components/v2/V2Create/ProjectPreview.tsx

```diff
@@ -55,7 +56,10 @@ export default function ProjectPreview({
     fundingCycleMetadata,
 
     distributionLimit: fundAccessConstraint?.distributionLimit,
-    distributionLimitCurrency: fundAccessConstraint?.distributionLimitCurrency,
+    distributionLimitCurrency:
```

Comment:
**question** why this change?

Reply:
Yeah this was lazy. Was seeing distributionLimitCurrency was coming in as 0 so it was fucking up some v2 stuff. So I changed the default redux state to use distributionLimitCurrency == V2_CURRENCY_ETH